packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -116,18 +121,14 @@ export function TopNavImpl(props: Props) {
   return (
     <div>
       <Menu theme="dark" mode="horizontal" selectable={false} className="ub-right" selectedKeys={[pathname]}>
-        <Menu.Item>
+        <Menu.Item style={{ paddingRight: '40px' }}>
```

Comment:
To give some space after the search form, since it was changed after the upgrade to v4.16.0